internal/hardware/hardware.go

```diff
@@ -23,6 +22,6 @@ func (s *Hardware) GetHardwareInformation() (*models.HardwareInfo, error) {
 	if err != nil {
 		return nil, err
 	}
-	log.Infof("Hardware info: %s", inventoryJson)
```

Comment:
Is this log entry not needed anymore?

Reply:
If remained, it is going to be printed multiple times to the log (on each retry). I don't think this information is useful on the agent side. And if reported properly, will be stored on the edge-device CR when succeeded to register.

Reply:
@masayag is right - that was the reason for the removal. That information isn't helpful being in the device-worker's logs.